node/lib/packaging/PackageInfo.ts

```diff
@@ -0,0 +1,173 @@
+
+/**
+ * Information about an OpenT2T package, as loaded from the package.json.
+ */
+export class PackageInfo {
```

Comment:
I'll add comments at the top to clarify that it's only meant to be a subset of package.json.

---
In reply to: [73277031](https://github.com/openT2T/opent2t/pull/2#discussion_r73277031) [](ancestors = 73277031,73275025)